scripts/setup-updates

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python3
+
+import sys
+import os
+import pydbus
+
+def symlink_updates(dest_dir, update_dir):
+    """Setup symlinks for the updates from the updates image.
+
+    :param str dest_dir: symlink target
+    :param str update_dir: symlink source (updates image content)
+    """
+    contents = os.listdir(update_dir)
+
+    for f in os.listdir(dest_dir):
+        dest_path = os.path.join(dest_dir, f)
+        update_path = os.path.join(update_dir, f)
+        if f in contents:
+            # recurse into directories, there might be files missing in updates
+            if os.path.isdir(dest_path) and os.path.isdir(update_path):
+                symlink_updates(dest_path, update_path)
+        else:
+            if f.endswith(".pyc") or f.endswith(".pyo"):
+                continue
+            os.symlink(dest_path, update_path)
+
+def setup_python_updates():
```

Comment:
Indentation 2 lines for top level functions please.

Reply:
Changed.